game/forms.py

```
@@ -11,42 +11,17 @@ class Meta(UserCreationForm.Meta):
-        enumeration through form-level error messages.
-        """
         email = self.cleaned_data.get('email')
         if email:
-            email = email.strip()
+            from django.contrib.auth import get_user_model
+            User = get_user_model()
+            if User.objects.filter(email__iexact=email).exists():
+                raise ValidationError(
+                    "A user with this email address already exists.",
+                    code='duplicate_email'
+                )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)